wee_slack.py

```diff
@@ -2605,6 +2605,23 @@ def process_message(message_json, eventrouter, store=True, **kwargs):
             channel.store_message(message, team)
         dbg("NORMAL REPLY {}".format(message_json))
 
+    download_location = config.get_string('files_download_location')
+    if not download_location:
+        return
+
+    for f in message_json.get('files', []):
+        if f.get('mode', '') == 'tombstone':
```

Comment:
You don't need the second argument to `.get` here.

Reply:
The comment was only for the specific line I commented on, i.e. `f.get('mode', '')`.